ci-operator/step-registry/openshift/e2e/test/qe/automated-release/openshift-e2e-test-qe-automated-release-chain.yaml

```diff
@@ -8,7 +8,8 @@ chain:
   - ref: cucushift-pre
   - ref: openshift-extended-test
   - ref: cucushift-e2e
+  - ref: openshift-extended-web-tests
```

Comment:
We should not add cypress test there, as this chain is also used by vsphere tests which are supposed not to run cypress tests.

Reply:
@liangxia Where do you suggest to add cypress test for auto release job?

Reply:
add directly in ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release.yaml ?

Reply:
Let's make sure cypress can run level0 test cases and make sure those level0 test cases are stable first.

Reply:
We only have several level0 tag cases. Remove the ref step in current pr, could add the web test step after we consider how to adapt it to different auto release jobs including vsphere.